pkg/util/provider/machinecontroller/machine_safety.go

```diff
@@ -331,10 +326,10 @@ func (c *controller) deleteMachineToSafety(obj interface{}) {
 		klog.Errorf("Couldn't convert to machine resource from object")
 		return
 	}
-	c.enqueueMachineSafetyOrphanVMsKey(machine)
+	c.enqueueMachineSafetyOrphanVMsKey()
```

Comment:
what is the purpose of calling a function which adds `""` to a rate limiting queue?

Reply:
whenever a machine is deleted , then we want to trigger an orphan collection, so we push an empty key to queue to do so. The key is just symbolic and is not used, it just acts like trigger.
Yes we could replace the function `enqueueMachineSafetyOrphanVMKey` with the single line called inside it.